corehq/apps/app_execution/static/app_execution/js/workflow_charts.js

```diff
@@ -1,102 +1,97 @@
+import "commcarehq";
+import nv from "jquery";
+import "moment/moment";
+import "d3/d3.min";
+import "nvd3/nv.d3.latest.min";  // version 1.1.10 has a bug that affects line charts with multiple series
```

Comment:
Is it the case that `d3` and `nv` (which were previously defined) are not referenced in this file? Given that there are no lint warnings about that I assume so.

Reply:
I'm surprised this didn't throw lint errors, those were used before. Fixed in abcbc2b46e0393c41d4e7150fc035d00fe3de060

(I did not test this PR because it's a superuser-only UI and the automation has been working well. But I should have, because I missed that the script doesn't work properly on files like this, where all of the dependency parameters are listed on the same line.)